modules/botAdmin.js

```diff
@@ -132,6 +179,23 @@ const Module = new Augur.Module()
     if (!reload) {
       u.errorLog.send({ embeds: [ u.embed().setDescription("Bot is ready!") ] });
     }
+    const requiredHidden = [
+      "../config/config",
+      "../config/rankConfig",
+      "../data/banned"
+    ];
+    for (const filename of requiredHidden) {
+      const prod = require(filename + ".json");
+      const repo = require(filename + "-example.json");
+      // console.log(`Checking ${filename}`);
+      const [m1, m2] = fieldMismatches(prod, repo);
+      if (m1.length > 0) {
+        u.errorHandler(Error("Mismatch from example file"), `Field(s) \`${m1.join("`, `")}\` in file ${filename + ".json"} but not example file`);
+      }
+      if (m2.length > 0) {
+        u.errorHandler(Error("Mismatch from example file"), `Field(s) \`${m2.join("`, `")}\` in example file but not ${filename + ".json"}`);
+      }
+    }
     const snowflakes = require("../config/snowflakes.json");
     Module.config.channels = snowflakes.channels;
     Module.config.roles = snowflakes.roles;
```

Comment:
Mixing these with the config isn't a great idea imo. Letting source files reference the snowflakes file when needed is a better solution.